amqp_job_queue.go

```diff
@@ -193,11 +194,20 @@ func (q *AMQPJobQueue) Jobs(ctx gocontext.Context) (outChan <-chan Job, err erro
 
 				err = json.Unmarshal(delivery.Body, &startAttrs)
```

Comment:
We fail to unmarshal `StartAttributes` (generated from user's .travis.yml), but we have successfully unmarshalled the `JobPayload` that contains job ID, so we don't need to extract job ID.